src/client/state/actions/index.js

```
@@ -1 +1,2 @@
 export { actionSelectStage as actionSelectStage } from "./stage";
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)